base_frame: "Archimede_base_link"
wheel_frame: "Archimede_fl_wheel"

drive_mode: "symmetric_ackermann"
max_wheel_speed: 10 #rad/s
wheel_radius: 0.085
wheel_width: 0.01

angle_crit: 0.58751447813558321651
angle_max: 1.6057029118347832

wheels_limits: 
  fl_wheel: [-1.6057029118347832, 0.58751447813558321651]
  bl_wheel: [-0.58751447813558321651, 1.6057029118347832]
  fr_wheel: [-0.58751447813558321651, 1.6057029118347832]
  br_wheel: [-1.6057029118347832, 0.58751447813558321651]
